cli/deno_dir.rs

```diff
@@ -312,19 +312,16 @@ impl DenoDir {
     // Fetch remote file and cache on-disk for subsequent access
     // not cached/local, try remote.
     let module_url_ = module_url.clone();
-    Either::B(self
+    Either::B(
+      self
         .fetch_remote_source_async(&module_url)
         // TODO: cache fetched remote source here - `fetch_remote_source` should only fetch with
         // redirects, nothing more
```

Comment:
Just to confirm, this TODO still applies, right?

Reply:
Still fiddling with API, I'll address it before it's landable